doc/GETTING_STARTED.md

```diff
@@ -126,7 +126,7 @@ Looking at the [available stream multiplexing](./CONFIGURATION.md#stream-multipl
 You can install `@libp2p/mplex` and add it to your libp2p node as follows in the next example.
```

Comment:
This and the Multiplexing section above makes references to `@libp2p/mplex` which are actually going to be outdated, we should update this to yamux